pkg/agent/handler/accessrequest.go

```diff
@@ -107,7 +107,9 @@ func (h *accessRequestHandler) onPending(ctx context.Context, ar *management.Acc
 
 	// check the application status
 	if app.Status.Level != prov.Success.String() {
```

Comment:
@jcollins-axway , well the code only checks for Success... so the other options are Error or Pending.  I think in Leor's case it was "stuck" in pending... i think because he had health check issues at the time he tried to request access.  So, unless we check the status level for error or processing?

Reply:
Can we have multiple checks then? If the app is Error return error saying app creation failed, only mitigation is to recreate the app/access request.

If pending then we have another issue where the app was not handled and not able to update its status, that is an interesting scenario that should not happen, but seems to have in this case.

Reply:
break it up... yep, can definitely do!

Reply:
may need something similar for credentials?